internal/api/handlers.go

```diff
@@ -360,6 +361,34 @@ func PostUserAccount(env *Environment) http.HandlerFunc {
 	}
 }
 
+func ChangeUserAccountPassword(env *Environment) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		id := r.PathValue("id")
+		var user certdb.User
+		if err := json.NewDecoder(r.Body).Decode(&user); err != nil {
+			logErrorAndWriteResponse("Invalid JSON format", http.StatusBadRequest, w)
+			return
+		}
+		if user.Password == "" {
+			logErrorAndWriteResponse("Password is required", http.StatusBadRequest, w)
+			return
+		}
+		ret, err := env.DB.UpdateUser(id, user.Password)
```

Comment:
should we have validations like more than 8 character or at least one capital letter and symbol here?

Reply:
Sure, and done it at user creation too.

Reply:
8? We still in the 90's or something? :laughing:

Reply:
Isn't that good enough for a minimum requirement?
I increased the length of the generated password. @DanielArndt

Reply:
I think its fine fyi :D